src/main/resources/application.yaml

```diff
@@ -24,8 +24,23 @@ spring:
           lob:
             non_contextual_creation: true
   jackson.date-format: yyyy-MM-dd
-  flyway.enabled: ${FLYWAY_ENABLED:true}
-  flyway.locations: classpath:db/migration,classpath:db/postgres
+  flyway:
+    enabled: ${FLYWAY_ENABLED:true}
+    locations: classpath:db/migration,classpath:db/postgres
+
+  mail:
+    from: no-reply@reform.hmcts.net
+    to:  sushant.choudhari@hmcts.net
```

Comment:
shushant id needs to be removed from here

Reply:
Should we put Praveen mail id in aat also?

Reply:
done